editor/create_dialog.h

```diff
@@ -32,11 +32,13 @@
 #define CREATE_DIALOG_H
 
 #include "editor/editor_help.h"
-#include "scene/gui/button.h"
 #include "scene/gui/dialogs.h"
-#include "scene/gui/item_list.h"
-#include "scene/gui/line_edit.h"
-#include "scene/gui/tree.h"
+
+// class Button;
```

Comment:
remove this and probl other comments if any left.

Reply:
I did not see any other comments when going over changes. It should still be removed though.